.github/workflows/benchmark.yml

```diff
@@ -40,10 +40,8 @@ jobs:
         run: |
           export NUM_CORES=`python -c "import joblib; print(joblib.cpu_count())"`
           export OMP_NUM_THREADS=$NUM_CORES
-          # Avoid threads over-subscription in parallel implementation
-          # TODO: remove once threadpoolctl is set.
-          export MKL_NUM_THREADS=1
-          export OPENBLAS_NUM_THREADS=1
+          export MKL_NUM_THREADS=$NUM_CORES
+          export OPENBLAS_NUM_THREADS=$NUM_CORES
```

Comment:
Now that you use threadpoolctl you should be amble to not set those variables anymore.